src/GeositeFramework/plugins/launchpad/templates.html

```diff
@@ -23,7 +30,14 @@
                 <li>
                     <div>
                         <h4><%= scenario.title %></h4>
-                        <p><%= scenario.description %></p>
+                        <div class="truncate-container">
+                            <% if (scenario.description.length > 140) { %>
+                                <p class="truncate-box"><%= scenario.description %></p>
+                                <button class="button truncate-toggle i18n" data-i18n="[title]See More" title="See More"></button>
```

Comment:
Not sure if this was intentional, but scenario buttons have a title attribute, but the plugin buttons do not.